package.json

```diff
@@ -30,6 +30,7 @@
     "eslint-plugin-jsdoc": "^2.3.1",
     "istanbul": "^0.4.3",
     "mocha": "^2.3.3",
+    "rewire": "^2.5.2",
```

Comment:
Haha nice 👍 

I would actually just monkey patch `fs.existsSync` but that's also ok 😁